src/FilterList.js

```diff
@@ -8,7 +7,7 @@ import sx from './sx'
 
 function ItemBase({children, count, theme, ...rest}) {
   return (
-    <a key={nanoid()} {...rest}>
+    <a {...rest}>
```

Comment:
Is the idea behind removing this that the consumer would set the key?

Reply:
That's right. Setting a key here doesn't actually do anything because we're not rendering an array from `ItemBase`. On top of that, `nanoid()` is random each time it runs. nanoid has a note about this in their readme:

> **Do not** call `nanoid` in the `key` prop. In React, `key` should be consistent among renders.
> 
> This is the bad example:
> 
> ```javascript
> <Item key={nanoid()} /> /* DON’T DO IT */
> ```